Formula/async-profiler.rb

```diff
@@ -0,0 +1,24 @@
+class AsyncProfiler < Formula
+  desc "Sampling CPU and HEAP profiler for Java"
+  homepage "https://github.com/jvm-profiling-tools/async-profiler"
+  url "https://github.com/jvm-profiling-tools/async-profiler/archive/v1.5.tar.gz"
+  sha256 "ffe8281f745a8e55bf9485c9ed9d30e5c4934174275cdb73b32e47cde35fffdd"
+
+  depends_on :java
+
+  def install
+    system "make"
+
+    libexec.install "build", "profiler.sh"
+
+    (bin/"async-profiler").write <<~EOS
+      #!/bin/bash
+      "#{Formula["async-profiler"].libexec}/profiler.sh" "$@"
+    EOS
+    chmod 0555, bin/"async-profiler"
+  end
+
+  test do
+    system bin/"async-profiler", "-v"
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
The only way we can properly exercise the app would be to start another Java process to attach to and get async-profiler to run a profile.

Are there any examples of tests for similar packages?